### PR TITLE
A Small Addition to Career Tests

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Retrying to add those career tests on Students and Graduates

## Additions

- Added another round of tests

## Changes

- Made a change to The-Bureau test to have it pass

## Testing

- Test following the instructions in the browser_tests README

## Review

- @anselmbradford 
- @sebworks 
- @jimmynotjim 

## Preview
image

[Preview this PR without the whitespace changes](?w=0)

## Notes

- I tried to add a linter, but it seemed out of scope for this PR and I'm running into problems making exceptions for global variables in Protractor
- We should add a task for bringing the other browser tests in line with most of our default for linting JS.

### CODE REVIEW

File: _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var StudentsAndGraduates = require( '../../page_objects/page_careers-students-and-graduates.js' );
+
+describe( 'Beta: Careers/Student-and-graduates', function() {
```
Comment (anselmbradford) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: Is describing this with `Beta: ` a convention? Minor, but technically this code is not on beta yet so is not describing a beta test.

**Comment by anselmbradford** [comment]
:+1: Just a minor question that doesn't necessarily have to be resolved in this PR.

File: _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js (hunk 1 above)
Comment (KimberlyMunoz) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: I'm matching what's on here: https://github.com/cfpb/cfgov-refresh/blob/flapjack/_tests/browser_tests/spec_suites/shared/shared_contact-us.js#L3

But I agree. Let's remove the "Beta." Mind if I grab all those in another PR?

Comment (KimberlyMunoz) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: I got another PR where I'm adding ESLint and fixing up consistency and linting errors like this one ready to go once this is in. :ticket:

Comment (anselmbradford) [inline _tests/browser_tests/spec_suites/shared/shared_careers-students-and-graduates.js]: :+1:

**Comment by jimmynotjim** [comment]
Tests pass and code looks good :+1:

**Comment by jimmynotjim** [comment]
Ha, that's what I get for leaving a PR open all day

**Pull request merged**